packages/api/src/api/client/people.ts

```diff
@@ -0,0 +1,26 @@
+import { Result } from "@formbricks/types/errorHandlers";
+import { NetworkError } from "@formbricks/types/errors";
+import { makeRequest } from "../../utils/makeRequest";
+import { TPerson, TPersonUpdateInput } from "@formbricks/types/people";
+
+export class PeopleAPI {
+  private apiHost: string;
+  private environmentId: string;
+
+  constructor(apiHost: string, environmentId: string) {
+    this.apiHost = apiHost;
+    this.environmentId = environmentId;
+  }
+
+  async updateAttribute(
```

Comment:
please rename to `update` to be consistent with the response method. The person update endpoint is now constructed in a way that it should be a general update of the person and in the future might update more than just attributes.